spec/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -154,6 +154,53 @@
   end
 end
 
+describe "rake After:fix_teen_and_up_imported_rating" do
+  let!(:noncanonical_teen_rating) { Rating.create(name: "Teen & Up Audiences", canonical: true) }
```

Comment:
You set this up as a canonical rating then `update_attribute` it to false later, I was confused until I see the before block.

I think a clearer way to do this is to skip validations while saving, all in the let block.

https://github.com/otwcode/otwarchive/blob/7a143f1d88a747e3550b7bd8a92c0140f7712f31/spec/lib/tasks/tag_tasks.rake_spec.rb#L77-L78